packages/server/src/middleware/errorHandler.ts

```diff
@@ -8,7 +8,7 @@ const errorHandler = (
   res: Response,
   next: NextFunction
 ) => {
-  const status = error.status || 500
+  const status = error.status || error.response.status || 500
```

Comment:
I don't know if this is the right way. But the problem was that `error.status` is undefined and error.response.status holds the right status 🤔 @mull

Reply:
I guess it's fine 🤷